TenPrintCover/C64Converter.m

```diff
@@ -0,0 +1,49 @@
+//
+//  C64Converter.m
+//  TenPrintCover
+//
+//  Created by Raman Singh on 2021-03-11.
+//  Copyright © 2021 NYPL_Labs. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "C64Converter.h"
+
+@implementation C64Converter
+
++ (NSString *)convertToC64:(NSString *)value {
+    
+    NSString *c64Letters = @" qQwWeErRtTyYuUiIoOpPaAsSdDfFgGhHjJkKlL:zZxXcCvVbBnNmM1234567890.";
+    
+    // returns a string with all the c64-letter available in the title or a random set if none
+    int i, len = (int)value.length;
+    NSMutableString *result = [NSMutableString stringWithCapacity:len];
+    char letter;
+    for (i=0; i<len; i++) {
+        letter = [value characterAtIndex:i];
+        NSRange range = [self indexOf:letter inString:c64Letters];
+        //        NSLog(@"letter: %c num: %d range: %d", letter, (int)letter, range.length);
+        if (range.length == 0) {
+            int anIndex = (int)(letter%c64Letters.length);
+            letter = [c64Letters characterAtIndex:anIndex];
+        }
+        [result appendString:[NSString stringWithFormat:@"%c", letter]];
+    }
+    return [NSString stringWithString:result];
+}
+
++ (NSRange) indexOf:(char) searchChar inString:(NSString *)string {
+    NSRange searchRange;
+    searchRange.location=(unsigned int)searchChar;
+    searchRange.length=1;
+    NSCharacterSet *set = [NSCharacterSet characterSetWithRange:searchRange];
+    if (set) {
```

Comment:
we should add a comment here explaining why we do this, since per the api this check looks unneeded. We should explain also how to trigger the "else" behavior (i.e. the accented characters.

Reply:
Done